ensure-issue-authorization/action.yml

```diff
@@ -0,0 +1,17 @@
+name: Ensure issue authorization
+description: Ensures issues are authorized and automatically closes unauthorized issues
+author: Karlo Nicholas Soriano
+inputs:
+  close-message:
+    description: Message to comment on the issue before closing
+  lock-message:
+    description: Message to comment on the issue before locking
+  lock-on-permission:
+    description: When user's permission level matches this, issue will be locked
+  close-on-permission:
+    description: When user's permission level matches this, issue will be closed
```

Comment:
What permission will you use for this? We're triggering on the lack of a write permission, so I don't know how this works? Does everyone get "read" by default?

Reply:
Everyone gets `read` by default according to this: https://help.github.com/en/articles/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization.